frontend/sozisel-app/src/components/utils/Button/Button.scss

```diff
@@ -1,14 +1,14 @@
-@use "../../../styles/color.scss";
+@import "../../../styles/styles";
 
 .button {
```

Comment:
Can we keep PascalCase for that scss classes? It looks strange when you have `.button` class as it looks almost as `button` tag but with an additional dot. Same with `.input`.

Reply:
ok, i will change it.